standalone/src/widget/index.tsx

```diff
@@ -92,7 +92,7 @@ async function initStoreWidget(store: Store, channelID: string) {
     } else {
         await getChannelMembers(channel.id)(store.dispatch, store.getState);
         const teams = getTeams(store.getState());
-        await selectTeam(Object.values(teams)[0])(store.dispatch, store.getState);
+        store.dispatch(selectTeam(Object.values(teams)[0]));
```

Comment:
actually we can change all the `selector(...)(store.dispatch, store.getState)` -> `store.dispatch(selector(...)), but for later.

Reply:
Is removing `async` going to create issues?

Reply:
nope -- store.dispatch is synchronous